### ISSUE TITLE
some project3.0 bugs

### ISSUE THREAD

**Issue by EbrahimKreem** [GroceriStar/food-datasets-csv-parser#236]
**Describe the bug**
errors in code linting and formating when I make a commit

**Screenshots**
![Screenshot from 2019-10-27 18-42-58](https://user-images.githubusercontent.com/32773511/67638397-fead7500-f8f4-11e9-8cfc-4b691d0880db.png)
![Screenshot from 2019-10-27 18-44-49](https://user-images.githubusercontent.com/32773511/67638398-00773880-f8f5-11e9-8b24-600f82f65826.png)

**Comment by atherdon** [comment]
yep i think we'll more similar errors soon, because i did a copy-paste with projects3.0 - and don't really care to check how it works. but most of them will be simple to fix.
at what project exactly it's generated?

**Comment by EbrahimKreem** [comment]
FAO
